.github/actions/setup/action.yml

```diff
@@ -98,4 +108,3 @@ runs:
           ~/.cargo/git/db/
           target/
         key: ${{ runner.os }}-${{ inputs.cargo-cache-key }}-${{ hashFiles('**/Cargo.lock') }}
-        restore-keys: ${{ runner.os }}-${{ inputs.cargo-cache-key }}
```

Comment:
Was this change intentional?

Reply:
Yes, this was to restore stale cache when there is no cache hit, which I don't think we want to do.